src/Components/Aspire.Qdrant.Client/QdrantClientSettings.cs

```diff
@@ -23,6 +23,19 @@ public sealed class QdrantClientSettings
     /// </summary>
     public string? Key { get; set; }
 
+    /// <summary>
+    /// Gets or sets a boolean value that indicates whether the Qdrant client health check is disabled or not.
+    /// </summary>
+    /// <value>
+    /// The default value is <see langword="false"/>.
+    /// </value>
+    public bool DisableHealthChecks { get; set; }
+
+    /// <summary>
+    /// Gets or sets a integer value that indicates the Qdrant health check timeout in milliseconds.
+    /// </summary>
+    public int? HealthCheckTimeout { get; set; }
```

Comment:
I know we already have MongoDB and Elasticsearch using `int?`, but I'm wondering if we should instead make these `TimeSpan` instead? That way we don't have to put "in milliseconds" and have people think it may be a different time unit.

MongoDB already shipped stable, but Elasticsearch is still in preview, so we could make that change. Then going forward, all new HealthCheckTimeout properties would be in terms of TimeSpan.